controllers/goharbor/jobservice/deployments.go

```diff
@@ -232,14 +241,31 @@ func (r *Reconciler) GetDeployment(ctx context.Context, jobservice *goharborv1.J
 		})
 	}
 
-	port := harbormetav1.JobServiceHTTPPortName
+	port := httpPort
+	portName := harbormetav1.JobServiceHTTPPortName
+
 	if jobservice.Spec.TLS.Enabled() {
-		port = harbormetav1.JobServiceHTTPSPortName
+		port = httpsPort
+		portName = harbormetav1.JobServiceHTTPSPortName
+	}
+
+	containerPorts := []corev1.ContainerPort{{
```

Comment:
Is it no longer supported to open both http and https ports?

Reply:
Yes, it will only open the port the jobservice listened on.